setup.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: page
+title: Flutter Setup
+nav_title: Flutter Setup
+permalink: /setup/
+---
+
+This guide describes how to set up your development environment to run Flutter apps on
+iOS or Android.
+
+[System requirements](#system_requirements)<br>
+[iOS setup](#ios_setup)<br>
+[Android setup](#android_setup)<br>
+[Get the Flutter SDK](#get_the_flutter_sdk)<br>
+[Atom editor](#atom_editor)
+
+## System requirements ##
+
+To install and run Flutter, your development environment must meet these minimum requirements.
+
+* **Operating Systems**: Mac or Linux (64-bit). Windows support is planned.
+* **Tools**: Flutter depends on these command-line tools being available on your environment.
+  * `bash`, `mkdir`, `rm`, `git`, `curl`, `unzip`
+
+## iOS setup ##
+
+To develop Flutter apps for iOS, you need a Mac with Xcode 7.2 or newer:
+
+1. Install Xcode 7.2 or newer (via [web download](https://developer.apple.com/xcode/) or
+the [Mac App Store](https://itunes.apple.com/us/app/xcode/id497799835)).
+2.  Make sure the Xcode EULA is signed by either opening Xcode once and confirming or
+running `sudo xcodebuild -license` from the command-line.
+
+With Xcode, you’ll be able to run Flutter apps on the iOS simulator.
+
+### Set up the iOS simulator ###
+
+To prepare to run and test your Flutter app on the iOS simulator, follow these steps:
+
+1. On your Mac, find the Simulator via Spotlight or by using the following command:
+
+    ```
+    $ open -a Simulator
+    ```
+
+2. Make sure your simulator is using a 64-bit device (iPhone 5s or later) by checking the settings
+in the simulator's **Hardware > Device** menu.
+3. Depending on your development machine's screen size, simulated high-screen-density iOS devices
+may overflow your screen. Set the device scale under the **Window > Scale** menu in the simulator.
+
+
+### Deploy to iOS devices ###
+
+To deploy your Flutter app to a physical iOS device, you’ll need some additional tools:
+
+1. Install [homebrew](http://brew.sh/).
+2. Open the terminal and run these command to install the toools for deploying Flutter apps to
+iOS devices.
+
+    <pre>
```

Comment:
Can you elaborate? When I try them here, \`\`\` seems to do the right thing:

   ```
   $ brew tap flutter/flutter
   $ brew install ideviceinstaller ios-deploy
   ```

   <pre>
   $ brew tap flutter/flutter
   $ brew install ideviceinstaller ios-deploy
   </pre>